voice/methods/about.md

```diff
@@ -27,3 +27,6 @@ Some of the URLs in this documentation contain placeholders for values that your
 | <code class="post">POST</code>     | [`/calls/{callId}/recordings/{recordingId}/transcription`](recordings/postCallsCallIdRecordingsRecordingIdTranscription.md)   | Request a transcription                                   |
 | <code class="get">GET</code>       | [`/calls/{callId}/recordings/{recordingId}/transcription`](recordings/getCallsCallIdRecordingsRecordingIdTranscription.md)    | Retrieve the transcription                                |
 | <code class="delete">DELETE</code> | [`/calls/{callId}/recordings/{recordingId}/transcription`](recordings/deleteCallsCallIdRecordingsRecordingIdTranscription.md) | Delete the transcription                                  |
+| <code class="put">GET</code>       | [`/conferences/{conferenceId}`](conferences/getConferencesConferenceId.md)                                                    | Retrieve conference information                           |
+| <code class="put">POST</code>      | [`/conferences/{conferenceId}`](conferences/postConferencesConferenceId.md)                                                   | Update an active conference                               |
+| <code class="put">GET</code>       | [`/conferences/{conferenceId}/members/{memberId}`](conferences/getConferenceMember.md)                                        | Retrieve information about a conference member            |
```

Comment:
Spacing is off. Not a big deal, but I think it's nice to keep it clean.
![2020-06-23-161252_1643x291_scrot](https://user-images.githubusercontent.com/2084330/85430424-a0ae5b80-b56f-11ea-8176-7607e48b2b32.png)